pom.xml

```
@@ -8,11 +8,11 @@
     </parent>
     <groupId>org.netbeans.contrib.yenta</groupId>
     <artifactId>parent</artifactId>
-    <version>1.2-SNAPSHOT</version>
+    <version>1.2</version>
```

revert please (and in child POMs)
```suggestion
    <version>1.2-SNAPSHOT</version>
```